tools/docs/manager.ts

```diff
@@ -14,8 +25,78 @@ function getManagerLink(manager: string): string {
   return `[\`${manager}\`](${manager}/)`;
 }
 
+function stringifyIssues(items: ItemsEntity[]): string {
+  if (!items) {
+    return '';
+  }
+  let list = '';
+  for (const item of items) {
+    list += ` - ${item.title} [#${item.number}](${item.html_url})\n`;
+  }
+  return list;
+}
+
+function extractIssues(
+  managerIssuesMap: Record<string, ManagerIssues>,
+  items: ItemsEntity[]
+): void {
+  if (!items || !managerIssuesMap) {
+    return;
+  }
+  for (const item of items) {
+    const type = item.labels
+      .find((l) => l.name.startsWith('type'))
+      ?.name.split(':')[1];
+    if (!type) {
+      continue;
+    }
+    const manager = item.labels
+      .find((l) => l.name.startsWith('manager'))
+      ?.name.split(':')[1];
+    if (!manager) {
+      continue;
+    }
+    if (!managerIssuesMap[manager]) {
+      managerIssuesMap[manager] = { bugs: [], features: [] };
+    }
+    switch (type) {
+      case 'bug':
+        managerIssuesMap[manager].bugs.push(item);
+        break;
+      case 'feature':
+        managerIssuesMap[manager].features.push(item);
+        break;
+      default:
+        break;
+    }
+  }
+}
+
+export async function getManagersGitHubIssues(): Promise<
+  Record<string, ManagerIssues>
+> {
+  const q = `repo:renovatebot/renovate type:issue is:open -label:priority-5-triage`;
+  const per_page = 100;
+  const managerIssuesMap: Record<string, ManagerIssues> = {};
+  try {
+    let previouslyFound = per_page;
+    for (let page = 1; previouslyFound === per_page; page++) {
+      const query = getQueryString({ q, per_page, page });
+      const res = await got(gitHubApi + query).json<GithubApiQueryResponse>();
```

Comment:
why not using `GithubHttp` class?

we could add a host rule with `process.env.GITHUB_TOKEN` when available (on CI)